web_src/js/features/projects.js

```diff
@@ -3,34 +3,46 @@ export default async function initProject(csrf) {
     return;
   }
 
-  const {Sortable} = await import(
-    /* webpackChunkName: "sortable" */ 'sortablejs'
+  const { Sortable } = await import(
```

Comment:
There was a recent linting change that forbids these spaces, should be auto-fixable using `npx eslint --fix  web_src/js/features/projects.js`.